s14/yield.rb

```diff
@@ -0,0 +1,77 @@
+def pass_control
+  puts 'This is inside the method'
+  yield   # Pass control from method to the block
+  puts 'Now I\'m back inside the method'
+end
+
+pass_control { puts 'I\'m very handsome'}
```

Comment:
What happens if you don't pass a block to the function `pass_control`?

Reply:
The yield simply receives nothing and continues with the next line.

Reply:
Please take a look at the documentation of `yield`.